magpy/stream.py

```diff
@@ -4109,6 +4109,14 @@ def fit(self, keys, **kwargs):
                 f_fit = self.harmfit(nt, val, fitdegree)
                 # Don't use resampled list for harmonic time series
                 x = nt
+            elif len(val)>1 and fitfunc == 'linear least-squares':
```

Comment:
This is checking for `'linear least-squares'`, but the method comment asks for `least-squares`.  You could translate from the gui option to the method option in the same place other gui options are translated:
https://github.com/usgs/magpy/pull/34/files#diff-5d0c7054373b573e8f63c12f95a6d3b7R3761